src/routes/docs/products/auth/security/+page.markdoc

```diff
@@ -39,4 +39,8 @@ Appwrite uses a [permissions model](/docs/advanced/platform/permissions) coupled
 With all Appwrite services, including databases and storage, access is granted at the collection, bucket, document, or file level.
 These permissions are enforced for client SDKs and server SDKs when using JWT, but are ignored when using a server SDK with an API key.
 
-{% partial file="auth-security.md" /%}
\ No newline at end of file
+{% partial file="auth-security.md" /%}
+
+# Session alerts {% #session-alerts %}
```

Comment:
Let's move this section into the `auth-security.md` partial because it can fit in both the pages where the partial is used.